### DISCUSSION TITLE
Adding context2d.arcTo() to Taxi Layout

### DISCUSSION THREAD

**Discussion by hassniazi** [cytoscape/cytoscape.js#3135] [Show and tell]
I'm often involved in producing architectures using a hierarchical layout like Taxi is ideal. Purely for aesthetic reasons, it would be preferable to add a radius to a 90 degree taxi bend, and fits in with many architecture design systems. Is it possible to add this feature to the Taxi layout please.

**Comment by hassniazi** [comment]
The aesthetic look I'm trying to replicate in cytoscape looks like this example 
![image](https://github.com/cytoscape/cytoscape.js/assets/61315137/25a75161-2223-4e75-bb1c-edc5a450cfb5)
![image](https://github.com/cytoscape/cytoscape.js/assets/61315137/5d7130e0-215a-446d-99bb-65c6a97b3777)

**Comment by maxkfranz** [comment]
That sounds like a great idea for a PR.

**Reply by hassniazi** [reply]
We've raised a PR [here](https://github.com/cytoscape/cytoscape.js/pull/3142). This is how the edge renders with this in place.

![image](https://github.com/cytoscape/cytoscape.js/assets/61315137/1f986cc4-0198-4502-994d-29be469b0ba9)


 Happy to adjust accordingly
